.github/workflows/go-pull-request.yml

```
@@ -35,7 +35,7 @@ jobs:
           format: 'sarif'
           output: 'trivy-results.sarif'
           severity: 'HIGH,CRITICAL'
-          skip-dirs: '.git,build,coverage,docs,sdks/java,sdks/cpp/unittests'
+          skip-dirs: '.git,build,coverage,docs,sdks/java,sdks/cpp/'
```

```suggestion
          skip-dirs: '.git,build,coverage,docs,sdks/java,sdks/cpp/,unittests'
```
still need to skip the unittests dir, even though there are no go unit tests